guest-tools/direct-boot/boot_uki.sh

```diff
@@ -46,15 +46,13 @@ EOM
 
 cleanup() {
     echo "cleanup ..."
```

Comment:
If you're deleting the functional part of the cleanup, is there a reason you're keeping the function that just echoes?

Reply:
thanks, i will remove it

Reply:
Done 57bef18762575d7cfbabc876ec221d46df674c1e